rllib/connectors/agent/pipeline.py

```diff
@@ -19,10 +21,12 @@
 class AgentConnectorPipeline(ConnectorPipeline, AgentConnector):
     def __init__(self, ctx: ConnectorContext, connectors: List[Connector]):
         super().__init__(ctx, connectors)
+        self.timers = defaultdict(_Timer)
 
     def reset(self, env_id: str):
         for c in self.connectors:
             c.reset(env_id)
+        self.timers.clear()
```

Comment:
hmm, do we want per-episode timer actually.
maybe just keep a global counter?

Reply:
Changed, thanks!